docs/features/PROMPTS.md

```diff
@@ -40,7 +40,7 @@ for adj in adjectives:
             print(f'a {adj} day -A{samp} -C{cg}')
 ```
 
-It's output looks like this (abbreviated):
+Its output looks like this (abbreviated):
```

Comment:
is there a reason this apostrophe was removed?

Reply:
@mauwii It's is a short note of `It is` implying reference to the state of the subject while `Its` implies ownership to the subject at hand. Two different things.

Reply:
thx - last english lessons are maybe too long ago 😅

Reply:
It's = "It is"
Its = possessive

I figured while I was in that directory, I'd fix this grammar issue.